src/fides/api/service/connectors/mongodb_connector.py

```diff
@@ -42,8 +42,7 @@ def build_uri(self) -> str:
 
     def create_client(self) -> MongoClient:
         """Returns a client for a MongoDB instance"""
-        config = MongoDBSchema(**self.configuration.secrets or {})
-        uri = config.url if config.url else self.build_uri()
+        uri = (self.configuration.secrets or {}).get("url") or self.build_uri()
```

Comment:
We try to get the url from the secrets directly so we don't get a validation error passing a dict with only a url through a schema

Reply:
i like this, clever little workaround

Reply:
i like this, clever little workaround